test/feature_discovery_test.dart

```diff
@@ -134,6 +197,12 @@ void main() {
 
         bool triggered = false;
 
+        // The surface size is set to ensure that the minimum overlay background size
+        // does not cover the button, but the content does.
+        await (TestWidgetsFlutterBinding.ensureInitialized()
+                as TestWidgetsFlutterBinding)
+            .setSurfaceSize(Size(3e2, 4e3));
```

Comment:
This can be `const`. @creativecreatorormaybenot You still haven't told me how you found these values!

Reply:
@axel-op Well, it was a bit of trial of error for sure, but you can check the "Clarification" commit for an explanation. I hope that it somewhat makes sense.

Maybe we should add linting for possible `const`s for example?

Reply:
Do you have an idea why this might happen: https://github.com/axel-op/feature_discovery/pull/38/checks?check_run_id=243815201#step:1:6

Reply:
> @axel-op Well, it was a bit of trial of error for sure, but you can check the "Clarification" commit for an explanation. I hope that it somewhat makes sense.
> Maybe we should add linting for possible consts for example?

Unfortunately I don't think the dartanalyzer is capable of such thing at the moment :(

Reply:
> Do you have an idea why this might happen: https://github.com/axel-op/feature_discovery/pull/38/checks?check_run_id=243815201#step:1:6

Yes, I edited my tag versions on my own action. I wondered if GitHub was capable of automatically detecting major versions based on the tags, but this showed me that it isn't...

Reply:
> Unfortunately I don't think the dartanalyzer is capable of such thing at the moment :(

Yes, it is. I will merge this PR and create one that adds some basic linting. We can always go further or exclude linting rules 👍

Reply:
https://github.com/axel-op/feature_discovery/pull/40#issue-323508303